scout/server/blueprints/cases/templates/cases/case.html

```diff
@@ -14,6 +14,30 @@
 {% block css %}
 {{ super() }}
 <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-select@1.13.9/dist/css/bootstrap-select.min.css">
+<style>
```

Comment:
This code is not new, I've just moved here because it makes more sense to have it under the css styles instead of under the javascript code